### PR TITLE
Pointcloud blob

### PR DESCRIPTION
**Pull request by ctieben**

### CODE REVIEW

File: include/sempr/entity/spatial/PointCloud.hpp [hunk 1]
```diff
@@ -104,7 +104,7 @@ typedef boost::variant< Channel<int8_t>,    // shall be used for boolean values
  * @brief The PointCloud class is a Entity that represents a Pointcloud
  * Currently stored as text - shall be a bin blob
  */
-class PointCloud : public Collection /*,public AbstractPointCloud<double> */
+class PointCloud : public Entity /*,public AbstractPointCloud<double> */
```
Comment (niniemann) [inline include/sempr/entity/spatial/PointCloud.hpp]: @ctieben I thought you implemented the "Collection" class as an interface? Is that not needed anymore?

**Comment by niniemann** [comment]
The last commit in this PR only adds a newline? :sweat_smile:

**Comment by ctieben** [comment]
And with this new line, the CI works ^^

File: include/sempr/entity/spatial/PointCloud.hpp (hunk 1 above)
Comment (ctieben) [reply include/sempr/entity/spatial/PointCloud.hpp]: The Collection is still an interface (as far as ODB allows it, at least). It derived from geometry and is an interface for GeometryCollection and MultiPoint. See a13e87f2aff910ec64bf0921afa723999c77c8c6
By this change the PointCloud is no Geometry anymore. It is an entity that contains a geometry (MultiPoint).

**Pull request merged**